pkg/kv/kvserver/batcheval/cmd_scan_test.go

```diff
@@ -231,6 +231,124 @@ func TestScanReverseScanWholeRows(t *testing.T) {
 	})
 }
 
+// TestScanReturnRawMVCC tests that a ScanRequest with the
+// ReturnRawMVCC option set should return the full bytes of the
+// MVCCValue in the RawBytes field of returned roachpb.Values.
+func TestScanReturnRawMVCC(t *testing.T) {
```

Comment:
Mildly surprised to see a test that "does it all over again" even though you also have one for "Get". Maybe premature optimization but if you also feel deduplication would be straightforward and warranted I would welcome some.

Reply:
Moved the get test into this one.